test/onnx/test_fx_op_consistency.py

```diff
@@ -67,9 +133,280 @@
 #     are now fixed, removed the corresponding xfail.
 #     2b. If a test is not failing consistently, use skip.
 EXPECTED_SKIPS_OR_FAILS: Tuple[onnx_test_common.DecorateMeta, ...] = (
+    skip(
+        "acos", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
+        reason=onnx_test_common.reason_onnx_does_not_support("Acos")
+    ),
+    skip(
+        "acosh", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
```

Comment:
@justinchuby I see it appears there are quite a few of these cases that makes a new casting problem/rule of its own. What's the right term to call it? "implicit integral promotion" (slightly modified from suggestion from chatgpt)?

Reply:
Good catch - I think pytorch may have different promotion policies? Would this fit into one of them?

Reply:
A quick check shows this logic is hardcoded https://github.com/pytorch/pytorch/blob/0be36d798ba959bfda6c448fc4832b5691df6e61/aten/src/ATen/TensorIterator.cpp#L835-L836

https://github.com/pytorch/pytorch/blob/0be36d798ba959bfda6c448fc4832b5691df6e61/aten/src/ATen/native/BinaryOps.cpp#L110-L112

Something we can induct for now, but would make a good rfc to propose adding that info into `native_functions.yaml`.